applications/chompy/main.tf

```diff
@@ -24,7 +24,8 @@ resource "heroku_app" "app" {
   }
 
   config_vars = {
-    IMPORTER_API_KEY = "${random_string.importer_api_key.result}"
+    CALLPOWER_API_KEY = "${random_string.importer_api_key.result}"
+    SOFTEDGE_API_KEY = "${random_string.importer_api_key.result}"
```

Comment:
We'll want these to be different, so let's rename `importer_api_key` to `random_string.callpower_api_key` above & then create a new `random_string.softedge_api_key` to generate the secret for `SOFTEDGE_API_KEY`.

Reply:
ah sorry about that @DFurnes ! I thought that would be called twice and generate two different strings each time. Updated!